draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -191,7 +191,7 @@ wire format used in {{RFC7858}}.
 Also note that while {{RFC1035}} says "Messages carried by UDP are
 restricted to 512 bytes", that was later updated by {{RFC6891}},
 and this protocol does not put any limit on the size of DNS
-on-the-wire format payloads.
+on-the-wire format payloads beyond what is already in {{RFC1035}}.
```

Comment:
The obvious reading of that is to substitute "what is already in 1035" with "restricted to 512 bytes", which I'm sure is not what you intend here.

I don't think that you need to make this change.  The only change I might consider is "put any **additional** limit on the size"